Examples/Java/Sources/Everything.java

```diff
@@ -25,30 +26,6 @@
 import java.util.Objects;
 import java.util.Set;
 
-interface EverythingMapPolymorphicValuesMatcher<R> {
```

Comment:
You can't have two top-level items in a Java class file. The interface has to be moved into the class.